kit/src/main/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMIntrinsic.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2014, Oracle and/or its affiliates.
+ * Copyright (c) 2012, 2022, Oracle and/or its affiliates.
```

Comment:
To keep the scope of the PR (which is already complex enough), please undo the changes in classes that only affect formatting.

Reply:
Reverted to previous state.